core-services/prow/02_config/_plugins.yaml

```diff
@@ -2233,6 +2239,12 @@ config_updater:
         - ci
       gzip: true
       name: job-config-4.0
+    ci-operator/jobs/stackrox/stackrox/*-release-4.1__ocp*.yaml:
+      clusters:
+        app.ci:
+          - ci
+      gzip: true
+      name: ci-operator-4.1-configs
```

Comment:
```suggestion
      name: job-config-4.1
```

Reply:
Oh yeah, I messed that one up through duplication. Let me try with `job-config-4.1`